scripts/tests-find.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
 
-ROOT=$(cd "$(dirname "$0")/.."; pwd)
+ROOT=$(
+  cd "$(dirname "$0")/.."
+  pwd
+)
 
-FILES=$(find $ROOT/{apps,packages,internal,templates} -name '*.test.ts')
+FILES=$(find $ROOT/{apps,packages,internal,templates,scripts} -name '*.test.ts')
```

Comment:
The final solution must be a GitHub action that all other repos can reuse.

Can you please port this PR here?
- https://github.com/FuelLabs/github-actions

Once there, this PR should simply integrate the action from `github-actions` into `fuels-ts`.

Ultimately, this PR will be the reference for integrating the action on other repos.

Reply:
https://github.com/FuelLabs/github-actions/pull/40